packages/compass-preferences-model/src/preferences.ts

```diff
@@ -336,22 +335,6 @@ const modelPreferencesProps: Required<{
     },
     deriveValue: deriveNetworkTrafficOptionState('enableMaps'),
   },
-  /**
-   * Switch to enable/disable error reports.
-   */
-  trackErrors: {
-    type: 'boolean',
-    required: true,
-    default: false,
-    ui: true,
-    cli: true,
-    global: true,
-    description: {
-      short: 'Enable Crash Reports',
-      long: 'Allow Compass to send crash reports containing stack traces and unhandled exceptions.',
-    },
-    deriveValue: deriveNetworkTrafficOptionState('trackErrors'),
-  },
```

Comment:
We only removed bugsnag earlier this year. Removing this from the model means that users who go back from a current Compass version to an older one won’t have this setting preserved – that’s probably fine?

Reply:
Compass that supports bugsnag is already a decent amount of versions ago. There is actually another preference that we already deleted from the model in the same way - agreedToLicense. I would say this is ok. I guess we could add a migration to always set these settings to the strict value, but I am not sure about that.